### PR TITLE
Add repos for kuksa.val.services migration

### PR DESCRIPTION
**Pull request by erikbosch**
Intended for migration of https://github.com/eclipse/kuksa.val.services/

- hvac_service and seat_service to be moved to kuksa-incubation. They are considered as examples and should not be used for production purposes. For kuksa-incubation we intend to not have dependabot alerts or PRs, to minimize maintenance needs.
- mock_service intended to be migrated to kuksa-mock-provider
- integration_test intended to be moved to kuksa-integration-test

Not to be merged until approved by @SebastianSchildt 

FYI: @lukasmittag @mikehaller

### CODE REVIEW

**Comment by netomi** [comment]
when you create PRs you could already add the people from whom you would like approval in the list of reviewers. We will wait till they approve. Then you can also mark the PR as ready so validation (and other checks) can be performed.

**Comment by erikbosch** [comment]
> when you create PRs you could already add the people from whom you would like approval in the list of reviewers. We will wait till they approve. Then you can also mark the PR as ready so validation (and other checks) can be performed.

I believe this repo has a different setup compared to other eclipse-kuksa repos. For other eclipse-kuksa repos I can both add reviewers and do a merge. In this repo I do not have the cogwheel to add additional reviewers

image

**Comment by netomi** [comment]
ah you dont have triage permissions yet as committer to be able to do that. That change was already merged, so will be live soon I guess. For now I can add the people as reviewers. This repo is indeed special, as you dont have write permissions to it as to the other repos in the organization.

**Comment by SebastianSchildt** [comment]
@erikbosch  agree wiht kuksa-mock-provider and kuksa-incubation,

not so sure about "integration test", as far as I udnerstood them, those ae not very "generic", but there is stuff there tightly coupled to things like seat service or someip service. So I would expect tests that are still useful going into the repositories of the the respective compnents, and not somewhere separate?

**Comment by erikbosch** [comment]
@SebastianSchildt - continuing detailed discussion on integration test in https://github.com/eclipse/kuksa.val.services/issues/119

**Comment by erikbosch** [comment]
@SebastianSchildt -removed the integration test repo. We can anyway add it later if needed/wanted

**Review by SebastianSchildt — APPROVED** [review]
Lgtm ⏫

**Review by netomi — APPROVED** [review]

**Pull request merged**
